MapboxDirections/MBRouteOptions.swift

```diff
@@ -319,6 +331,10 @@ open class RouteOptions: NSObject, NSSecureCoding {
             params.append(URLQueryItem(name: "roundabout_exits", value: String(includesExitRoundaboutManeuver)))
         }
         
+        if !excludedRoadClasses.isEmpty {
+            params.append(URLQueryItem(name: "exclude", value: excludedRoadClasses.description))
```

Comment:
This would be a good place to assert that only one option is specified in `roadClassesToAvoid`.